frontend/src/components/dropzone/DropzoneComponent.vue

```diff
@@ -113,7 +179,7 @@ const dropzoneClasses = computed(() => {
             'group-hover:text-blue': props.variant === 'blue',
           }"
         >
-          Glissez et déposez votre fichier ou cliquez ici
+          Drag and drop your file or click here
```

Comment:
```suggestion
          Glissez et déposez votre fichier ou cliquez ici
```
On est en français partout